ngw.tf

```diff
@@ -0,0 +1,23 @@
+locals {
```

Comment:
Do not separate NAT resources from `public.tf`.
NAT Gateway will never be used with `private` subnet.